fuzz/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "ruff-fuzz"
+version = "0.0.0"
+authors = [
+  "Addison Crump <research@addisoncrump.info>",
+]
+publish = false
+edition = "2021"
```

Comment:
Nit: You can inherit the edition from the workspace
```suggestion
edition = { workspace = true }
```

Reply:
Actually, not so. Fuzz needs to be in its own workspace and cannot import from the parent workspace, as the build uses separate RUSTFLAGS.